### DISCUSSION TITLE
How can i know the audio's playing status of the webpage?

### DISCUSSION THREAD

**Discussion by wu-zhi-qiang** [cefsharp/CefSharp#4320] [General]
how can i know the audio's playing status when i playing a audio or video?
or when the audio play begin , when the audio play pause, when the audio play stop.

PS:
I implate IAudioHandler, when i reutrn true at GetAudioParameters, i can get the OnAudioStreamStarted, OnAudioStreamStopped callback, but there will be no sound in speaker.
I want only get the callback of OnAudioStreamStarted,OnAudioStreamStopped, don't want capture the audio.

waiting for your help. thank you.

**Comment by amaitland** [comment]
I'm not aware of an API specifically for this purpose. You can review the CEF API online at https://cef-builds.spotifycdn.com/docs/107.1/index.html

CEF has it's own feature request forum at https://www.magpcss.org/ceforum/viewforum.php?f=7

**Comment by BuslikDrev** [comment]
https://stackoverflow.com/questions/51259813/call-c-sharp-function-from-javascript-using-cefsharp-in-windows-form-app
